autoPyTorch/pipeline/components/training/losses.py

```diff
@@ -11,16 +11,17 @@
 from autoPyTorch.constants import BINARY, CLASSIFICATION_TASKS, CONTINUOUS, MULTICLASS, REGRESSION_TASKS, \
     STRING_TO_OUTPUT_TYPES, STRING_TO_TASK_TYPES, TASK_TYPES_TO_STRING
 
+
 losses = dict(classification=dict(
     CrossEntropyLoss=dict(
-        module=CrossEntropyLoss, supported_output_type=MULTICLASS),
+        module=CrossEntropyLoss, supported_output_types=[MULTICLASS, BINARY]),
     BCEWithLogitsLoss=dict(
-        module=BCEWithLogitsLoss, supported_output_type=BINARY)),
+        module=BCEWithLogitsLoss, supported_output_types=[BINARY])),
     regression=dict(
         MSELoss=dict(
-            module=MSELoss, supported_output_type=CONTINUOUS),
+            module=MSELoss, supported_output_types=[CONTINUOUS]),
```

Comment:
Can you change these dicts to NamedTuple or this [BaseDict](https://github.com/automl/Auto-PyTorch/pull/88/commits/ce1f63851c3fb1cc26db2a356d8990ada2045b28)?
By explicitly define the elements of dicts, the readability will increase and we can avoid typo.
The greatest benefit from those data structures is your working environment can suggest the keys.

Reply:
I can change it later, as BaseDict is included in your PR. is it fine?

Reply:
Yes, that is fine:)